string object_name
---
moveit_msgs/Grasp grasp_result